hibernate-ogm/playsport-demo/src/test/resources/back_to_5_1_0_Final_ogm_version.patch

```diff
@@ -0,0 +1,41 @@
+Index: hibernate-ogm/playsport-demo/pom.xml
+IDEA additional info:
```

Comment:
What is this file for?

Reply:
It is no longer necessary, I would only use the 5.2.0.Alpha tag. Ok?